lms/models/assignment.py

```diff
@@ -69,7 +69,7 @@ class Assignment(CreatedUpdatedMixin, Base):
     )
     """Whether this assignment is gradable or not."""
 
-    title = sa.Column(sa.Unicode, nullable=True)
+    title: Mapped[str | None] = mapped_column(sa.Unicode)
```

Comment:
This is not necessary, but checking that assignment.title can be null in DB, adapting to the new SQLA style.